zebra-chain/src/primitives/address.rs

```diff
@@ -14,17 +17,17 @@ pub enum Address {
 
     /// Sapling address
     Sapling {
-        /// Address' network
-        network: Network,
+        /// Address' network kind
+        network: NetworkKind,
```

Comment:
I think this enum is only used by the RPCs. Do we want to try using the network name here instead? There's no way to distinguish between testnet and regtest for transparent addresses, but there may be for Sapling/Orchard.